### PR TITLE
(main navigation) megamenu, collapsed nav, sticky nav

### PR DESCRIPTION
**Pull request by sophiaaachow**
# Description

Please include a summary of the change and where applicable, the issue you are addressing. Please also include relevant motivation and context. List any dependencies that are required for this change. If necessary, please update documentation (found under the `collections` folder) as well.

Fixes # (issue number)

## Type of change

Please check on the checkbox for those that are relevant (put x between the brackets)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Code enhancement and update (non-breaking change)
- [ ] Security patch

## How Has This Been Tested?

Please describe or list the test cases that you ran to verify your changes, and provide instructions so we can reproduce them.

### CODE REVIEW

File: collections/_components/main-navigation.html [hunk 1]
```diff
@@ -10,50 +10,194 @@
...
+                </a>
+                <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
+                    <li><a class="dropdown-item" href="">Action</a></li>
+                    <li><a class="dropdown-item" href="">Another Action</a></li>
+                    <li><a class="dropdown-item" href="">Something</a></li>
+                    <div class="dropdown-divider"></div>
+                    <li><a class="dropdown-item" href="">Seperated Link</a></li>
+                </ul>
+            </li>
+            <li class="nav-item sgds dropdown">
```
Comment (rav3) [inline collections/_components/main-navigation.html:39]:
```suggestion
            <li class="nav-item sgds dropdown has-megamenu">
```

File: collections/_components/main-navigation.html [hunk 2]
```diff
@@ -10,50 +10,194 @@
...
+                <a class="nav-link sgds dropdown-toggle" href="#" id="megamenu" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                Megamenu<i class="bi bi-chevron-down"></i>
+                </a>
+                <div class="dropdown-menu p-3" aria-labelledby="megamenu">
```
Comment (rav3) [inline collections/_components/main-navigation.html:43]:
```suggestion
                <div class="dropdown-menu megamenu p-3" aria-labelledby="megamenu">
```

**Review by clukhei — COMMENTED** [review]
Screenshot 2022-07-19 at 10 44 18 AM
 
will this be an issue? @rav3

**Comment by rav3** [comment]
@sophiaaachow 
for the default navigation example, you can add in dummy content like what u did to the sticky navigation example.

**Review by clukhei — APPROVED** [review]

**Pull request merged**